modules/node/src/test/deployContracts.ts

```diff
@@ -0,0 +1,184 @@
+import { EXPECTED_CONTRACT_NAMES_IN_NETWORK_CONTEXT as coreContracts } from "@connext/types";
+import { Contract, ContractFactory, Wallet } from "ethers";
+import { EtherSymbol, Zero } from "ethers/constants";
+import { formatEther, keccak256, parseEther } from "ethers/utils";
+import fs from "fs";
+
+const tokenArtifacts = require("openzeppelin-solidity/build/contracts/ERC20Mintable.json");
+
+const appContracts = ["SimpleLinkedTransferApp", "SimpleTransferApp", "SimpleTwoPartySwapApp"];
+
+console.log(`Core contracts: ${JSON.stringify(coreContracts)}`);
+
+const artifacts = {};
+for (const contract of coreContracts) {
+  try {
+    artifacts[contract] = require(`@connext/cf-adjudicator-contracts/build/${contract}.json`);
+    console.log(`Imported adjudicator contract: ${contract}`);
+  } catch (e) {
+    artifacts[contract] = require(`@connext/cf-funding-protocol-contracts/build/${contract}.json`);
+    console.log(`Imported funding contract: ${contract}`);
+  }
+}
```

Comment:
Fun fact: new work I've been doing in my CI/CD branch includes wrapping the ethprovider stuff in a docker image. That means no need to copy/paste this contract deployment stuff. Instead, we start up our ethprovider image (instead of default truffle/ganache:latest) & that will automatically deploy contracts on startup.